OrbitCore/SerializationMacros.h

```diff
@@ -3,4 +3,6 @@
 //-----------------------------------
 #pragma once
 
-#define ORBIT_SERIALIZABLE template <class Archive> void serialize(Archive & a_Archive, std::uint32_t const a_Version)
\ No newline at end of file
+#define ORBIT_SERIALIZABLE \
+  template <class Archive> \
+  void serialize(Archive& a_Archive, std::uint32_t const a_Version)
```

Comment:
I get this when trying to compile on linux (adding new line at the end of define fixes it), not sure why it was not here before formatting:
OrbitCore/SerializationMacros.h:6:28: warning: backslash-newline at end of file
 #define ORBIT_SERIALIZABLE \

Reply:
I don't get this on clang. anway, I just added the newline.